waiter/test/waiter/reporters_test.clj

```diff
@@ -0,0 +1,173 @@
+;;
+;; Copyright (c) Two Sigma Open Source, LLC
+;;
+;; Licensed under the Apache License, Version 2.0 (the "License");
+;; you may not use this file except in compliance with the License.
+;; You may obtain a copy of the License at
+;;
+;;  http://www.apache.org/licenses/LICENSE-2.0
+;;
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+;;
+(ns waiter.reporters-test
+  (:require [clj-time.core :as t]
+            [clojure.test :refer :all]
+            [metrics.core :as mc]
+            [metrics.counters :as counters]
+            [waiter.metrics :refer :all]
+            [waiter.reporter :as r])
```

Comment:
Let us use one name consistently, it is aliased to `reporter` elsewhere. Alternatively, we have been using `:refer :all` in unit tests of a particular namespace.